src/types.rs

```diff
@@ -10,6 +10,32 @@ use std::{
 
 use crate::execution::RuntimeModule;
 
+#[cfg(feature = "execution-trace")]
+#[derive(Debug, Clone, PartialEq)]
+pub enum AbiTraceType {
+    None,
+    Bool(bool),
+    U8(u8),
+    I32(i32),
+    I64(i64),
+    U64(u64),
+    F64(f64),
+    ByteArray(Vec<u8>),
```

Comment:
Bytes?

Reply:
Type names have already been defined with the explorer guys ...

Reply:
We can still change it today it's not a big change. Personally I  prefer ByteArray or ByteArrayArray. But current ones are ok for now.